src/algebra/module/zlattice.lean

```diff
@@ -86,14 +87,14 @@ begin
     ← (eq_int_cast (algebra_map ℤ K) _), basis.restrict_scalars_repr_apply, coe_mk],
 end
 
-lemma mem_fundamental_domain {x : E} :
-  x ∈ (fundamental_domain b) ↔ fract b x = x :=
+lemma mem_fundamental_domain_iff {x : E} :
```

Comment:
I think the previous name was better. The `_iff` is clear from the rest of the name already.
```suggestion
lemma mem_fundamental_domain {x : E} :
```